src/effects/How to make an effect stateless.txt

```diff
@@ -0,0 +1,189 @@
+4 STEPS TO MAKE AN EFFECT STATELESS
```

Comment:
If you think adding this memo to the source code tree will be a helpful reference to you and others, why not.  We should aim to delete this again when all effects have been migrated.